crm_stage_probability/tests/test_crm_lead_probability.py

```diff
@@ -47,6 +52,13 @@ def test_create_opportunity(self):
         default_stage = self.env["crm.stage"].browse(default_stage_id)
         self.assertEqual(opportunity.probability, default_stage.probability)
         self.assertFalse(opportunity.is_automated_probability)
+        # test if there is no default stage
+        self.env["crm.stage"].search([("fold", "=", False)]).write({"fold": True})
+        opportunity = self.env["crm.lead"].create(
+            {"name": "My opportunity", "type": "opportunity"}
+        )
+        self.assertEqual(opportunity.probability, 10)
+        self.assertFalse(opportunity.is_automated_probability)
 
     def test_create_opportunity_default_stage_id(self):
```

Comment:
Hi, @vib-adhoc !

Seem that the test needs to include a call to onchange_stage_id_values . So maybe a new method that invoques that could solve the problem. 
Maybe a def test_change_stage_id_values(self) with a simple event that calls onchange_stage_id_values

Thank you!

Reply:
Hi @pere-aquarian ! I added a test, thanks for your suggestion.

Reply:
thanks a lot, @vib-adhoc and @jjscarafia